src/JsonApiPaginateServiceProvider.php

```diff
@@ -36,11 +36,11 @@ protected function registerMacro()
             $numberParameter = config('json-api-paginate.number_parameter');
             $sizeParameter = config('json-api-paginate.size_parameter');
 
-            if (is_null($maxResults)) {
-                $maxResults = $configuredMaximum;
+            if (null !== $maxResults) {
```

Comment:
I'm not a fan of Yoda style comparisons. Could you revert this line to it's former glory?